db/database-init.sql

```diff
@@ -0,0 +1,25 @@
+\c badmckinney
+DROP DATABASE articles_and_products;
+CREATE DATABASE articles_and_products;
+
+\c articles_and_products
+
+CREATE TABLE articles (
+  id    SERIAL PRIMARY KEY,
+  title    varchar(90) UNIQUE,
+  author    varchar(45),
+  body    varchar(510)
+);
+
+CREATE TABLE products (
+  id    SERIAL PRIMARY KEY,
+  name    varchar(45) UNIQUE,
+  price    integer,
+  inventory    integer
+);
+
+INSERT INTO articles (title, author, body) VALUES ('Y2K', 'Bill Gates', 'Its the end of the world');
```

Comment:
Although these next 4 lines are good for testing, once you're ready to make a pull request, you should probably delete these lines.